packages/core/upload/admin/src/pages/App/MediaLibrary/components/Header.tsx

```diff
@@ -35,9 +59,10 @@ export const Header = ({
       })}
       subtitle={
         breadcrumbs &&
+        typeof breadcrumbs !== 'boolean' &&
         folder && (
           <Breadcrumbs
-            tag="nav"
+            tag="div"
```

Comment:
Why do you change this? I think nav is the correct tag for accessibility

https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/

Reply:
the value "nav" is not an accepted one by Breadcrumbs component in the DS

Reply:
You don't need to provide a tag prop. The design system handles it:

https://github.com/strapi/design-system/blob/107f3b86f96a6dfb080a0d4fedca366ef895bd5c/packages/design-system/src/components/Breadcrumbs/Breadcrumbs.tsx#L29